packages/server/test/support/helpers/e2e.coffee

```diff
@@ -315,16 +315,18 @@ module.exports = {
         ## if we have browser in the stdout make
         ## sure its legit
         if matches = browserNameVersionRe.exec(stdout)
-          [str, key, browserName, version, headless] = matches
+          [str, key, customPath, browserName, version, headless] = matches
 
-          if b = options.browser
-            expect(_.capitalize(b)).to.eq(browserName)
+          browser = options.browser
+
+          if browser and not customPath
+            expect(_.capitalize(browser)).to.eq(browserName)
```

Comment:
disregard my last comment if you have to have `customPath` here.

if you do keep it, rename it to `customBrowserPath`

Reply:
Gonna keep it as-is, I think that there will be more e2e tests that happen to use "custom" browsers in the future, so they should "just work" with this change being made